src/components/Item.jsx

```diff
@@ -17,6 +32,26 @@ const Item = ({ name, id, lastPurchasedDate }) => {
       return now - lastPurchasedDate.seconds < day;
     }
   };
+
+  const getPurchaseInterval = () => {
+    let lastEstimate = frequency;
+    let latestInterval =
+      lastPurchasedDate != null
+        ? Math.floor(
+            // convert from date to seconds
+            (new Date().getTime() / 1000 - lastPurchasedDate.seconds) /
+              // convert from seconds to day
+              (60 * 60 * 24),
+          )
+        : lastEstimate;
```

Comment:
Hi @Lukabaramishvili We were thinking to use `round`. But we thought if it's only 0.5 day, it shouldn't round up as 1 full day.

Reply:
Is it okay that if it's 0.99 day, that it gets recorded as zero?

Reply:
Hmm that’s true. So round is better? Begin to regret to not use a library that handles date time 🙈

Reply:
Haha, it depends on where you want to draw the line. In practice, it will probably be closer to the following if rounding.
- 6.4 -> 6
- 17.8 -> 18

Which whether you're rounding or using a floor function, nobody will likely notice. Rounding is probably more _correct_ over time though.

Reply:
Sounds fair. Alright, changed to `round ` now 👍

Reply:
Also, if I look at `estimate.js`, it doesn't necessarily say that it is expecting an integer. Passing it `1.8` or `0.99` seems to also be a valid representation of a day. What I don't know is if that would potentially throw something else off. Might be worth playing with though.

Reply:
I've done a small bit of testing, and it looks like you don't need to do any rounding at all. Passing in the raw value will work just fine. In face `calculateEstimate` will hand you back a rounded number. For accuracy, it's best to leave the rounding till the end. I'd recommend removing any rounding at all and just passing in the raw number that was calculated. 👍 

I had to read the code and test a bit more to be sure.

Reply:
I was thinking that `Math.ceil` could be a better solution here. Suppose it is a first day in the JS world and we pass in 0.3 of it - `round` and `floor` will round the number down and will return `0`. While `Math.ceil` will return `1`.

On the other hand, I do get what Kevin says in the above comment. It might be a good idea to leave the rounding till the end. _( But also let's keep it in mind )_